src/components/DownloadLoginFile.js

```diff
@@ -119,16 +119,18 @@ class DownloadLoginFile extends Nimiq.Observable {
      * @param {MouseEvent} event
      */
     _onMouseDown(event) {
+        /** @type {HTMLElement} */
+        const target = (event.target);
+        if (target.matches('.continue')) {
+            this._onDownloadEnd();
+            return;
```

Comment:
Is this needed?
The continue button already has a stopPropagation

Reply:
As discussed, this handling is for two different events: `mousedown` and `click`. I removed the `_onDownloadEnd()` call from this `mousedown` handler, as the `click` handler should cover both mobile and desktop usages of the Continue button.